src/services/sports/index.jsx

```
@@ -0,0 +1,400 @@
+              />
+            </div>
+          </div>
+        )}
+      </Form>
+    </>
+  );
+};
+export const EditPicTour = async () => {};
+export const DeletePicTour = async () => {};
+
+export const SportsPreviewer = ({ image, isOpen, onClose, isTop = false }) => {
```

ESLint

[error] 'image' is missing in props validation

(react/prop-types)